src/ScottPlot5/controls/ScottPlot.WPF/WpfPlot.xaml.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using ScottPlot.Control;
+using SkiaSharp;
+using SkiaSharp.Views.Desktop;
+using Key = ScottPlot.Control.Key;
+using MouseButton = ScottPlot.Control.MouseButton;
+
+namespace ScottPlot.WPF
```

Comment:
Do you think there's utility in creating an `IPlotControl`? It could contain things like `Plot`, `MouseCoordinates`, `Backend`, `GetInputState()`, `OnPaintSurface()`, `GetScottPlotKey()`, and might make it a little easier to implement new controls.

Not sure how much shared logic there is though, but a `ScottPlotControlBase` with virtual functions could be another way to guide new controls into implementing the necessary functionality

Not required for this PR - just thinking out loud

Reply:
I don't know, `Backend`, `GetInputState`, `OnPaintSurface`, and `GetScottPlotKey` are all `private`, so they can't be in the interface. Not to mention they're implementation details, they probably shouldn't be relied on as part of an interface's contract.

As for `Plot`, `Refresh`, and `MouseCoordinates`, I don't think they're bad things to guarantee across all controls, but frankly there's no reason for a control to implement the interface except for peer pressure. An interface is a great way to make code indifferent to the implementation of something, but if someone is on WPF they _need_ the WPF implementation.

Although, an interface for the `Backend` may be useful for testing, but I think we need to decide what testing will look like first.